pkg/cmd/cli/describe/projectstatus.go

```diff
@@ -1259,6 +1263,53 @@ func (l *secretLoader) AddToGraph(g osgraph.Graph) error {
 	return nil
 }
 
+type pvcLoader struct {
+	namespace string
+	lister    kclient.PersistentVolumeClaimsNamespacer
+	items     []kapi.PersistentVolumeClaim
+}
+
+func (l *pvcLoader) Load() error {
+	list, err := l.lister.PersistentVolumeClaims(l.namespace).List(kapi.ListOptions{})
+	if err != nil {
+		return err
+	}
+
+	l.items = list.Items
+	return nil
+}
+
+func (l *pvcLoader) AddToGraph(g osgraph.Graph) error {
+	for i := range l.items {
+		kubegraph.EnsurePersistentVolumeClaimNode(g, &l.items[i])
+	}
+
+	return nil
+}
+
+type nodeLoader struct {
```

Comment:
> I think most clusters will use multiple hosts anyway. Thoughts?

Yeah, most will have multiple nodes.  Remember this problem when we start building a graph API.  A graph API could compute markers based on knowledge of nodes that are never exposed to users.  The direction of edges becomes critical and has to flow from a node that provides awareness of the other node.